index.js

```diff
@@ -373,11 +367,19 @@ class WriteStreamSync extends WriteStream {
   }
 }
 
-const writev = (fd, iovec, pos, cb) => {
-  const done = (er, bw) => cb(er, bw, iovec)
-  const req = new FSReqWrap()
-  req.oncomplete = done
-  binding.writeBuffers(fd, iovec, pos, req)
+let writev = fs.writev
+/* istanbul ignore next */
+if (!writev) {
+  const binding = process.binding('fs')
+  const writeBuffers = binding.writeBuffers
```

Comment:
It doesn't look like this variable is being used anywhere.
```suggestion

```

Reply:
Thanks. Removed in a fixup commit.